Public/Src/Engine/UnitTests/FingerprintStore/Test.BuildXL.FingerprintStore.dsc

```diff
@@ -36,6 +36,6 @@ namespace Test.BuildXL.FingerprintStore {
         runtimeContent: [
             importFrom("BuildXL.Utilities.UnitTests").TestProcess.deploymentDefinition
         ],
-        runTestArgs: { weight: 10 },
+        runTestArgs: { weight: 2 },

```

Comment:
>  runTestArgs: { weight: 2 }, [](start = 6, length = 29)

Why not just remove them completely?

Reply:
Let's go with this weight first. If timeout doesn't happen, we can start reconsidering all weights that we set in our self-host.